typescript/sdk/src/deploy/HyperlaneDeployer.ts

```diff
@@ -258,6 +261,20 @@ export abstract class HyperlaneDeployer<
     );
   }
 
+  // prettier-ignore
+  private cacheContract<
+    K extends keyof Factories,
+  >(chain: Chain, contractName: K, contract: HyperlaneContract) {
+    if (!this.deployedContracts[chain]) {[]
+      this.deployedContracts[chain] = {};
+    }
+    if (this.deployedContracts[chain]) {
+      // TODO: This doesn't compile but it *does* work.
+      // Had to comment out to be able to push since prettier complains.
+      // this.deployedContracts[chain]?.[contractName as string] = contract;
```

Comment:
`@ts-ignore`?

Reply:
I tried that but *prettier* complains in our pre-commit hook

Reply:
Open to suggestions if you know how to do this in a way that makes the transpiler happy